### PR TITLE
MS. Douce 104: new description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

File: collections/Douce/MS_Douce_104.xml [hunk 1]
```diff
@@ -37,40 +36,238 @@
...
+                     <p>Suggested to be written in the dialect spoken by English settlers in Ireland by Angus McIntosh, M. L. Samuels, and Michael Benskin, <title>A Linguistic Atlas of Late Mediaeval English</title> (Aberdeen, 1986), 1:147, based on analysis from fols. 50r-79v, with a few additions from text to fol. 108v. Irish scribes were also active in London, but this possibility seems less likely based on the artistic style and text: see Pearsall, p. xiii.</p>
                   </origin>
-                  <provenance notAfter="1834" notBefore="1757" resp="#MMM">
-                     <persName key="person_69848690" role="fmo">Francis Douce, 1757–1834</persName>
-                  </provenance>
+                  <provenance><persName key="person_71740714" role="fmo">James Ley, first earl of Marlborough (1550–1629)</persName>, chief justice of king’s bench in Ireland: his arms on the former binding.</provenance>
+                  <provenance notAfter="1834" notBefore="1757" resp="#MMM"><persName key="person_69848690" role="fmo">Francis Douce (1757–1834)</persName>: his bookplate, notes (upper pastedown and fol. 33a), and sporadic line numbering.</provenance>
```
Comment (holfordm) [inline collections/Douce/MS_Douce_104.xml:264]: details on Douce's acquisition in Douce Legacy https://babel.hathitrust.org/cgi/pt?id=mdp.39015036877796&view=1up&seq=168&skin=2021

File: collections/Douce/MS_Douce_104.xml [hunk 2]
```diff
@@ -37,40 +36,238 @@
...
                   <acquisition resp="#MMM" when="1834">Bequeathed to the Bodleian in 1834</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>
-                           <ref target="https://medieval.bodleian.ox.ac.uk/about">Summary description</ref> abbreviated from the Summary Catalogue (1897). Decoration, localization and date follow Pächt and Alexander (1973). <listBibl>
+                        <source>Description by Andrew Dunning (August 2022), largely based on from the detailed discussions by Derek Pearsall, introduction to <title>Piers Plowman: A Facsimile of Bodleian Library, Oxford, MS Douce 104</title> (Cambridge, 1992); and Kathleen L. Scott, ‘<ref target="https://doi.org/10.1484/J.YLS.2.302913">The Illustrations of Piers Plowman in Bodleian Library MS. Douce 104</ref>’, <title>The Yearbook of Langland Studies</title> 4 (January 1990): 1–86, reprinted in <title>Piers Plowman: A Facsimile</title>. <listBibl>
```
Comment (holfordm) [inline collections/Douce/MS_Douce_104.xml:270]: typo ("based on from")

File: collections/Douce/MS_Douce_104.xml [hunk 3]
```diff
@@ -37,40 +36,238 @@
...
+                           <item>
+                              <label>(fol. 109v)</label> A proud priest: standing young man, tonsured, in a tight decorated jacket and hose, long shoes, a sword hanging at his front, gesturing and looking at the text.</item>
+                           <item>
+                              <label>(fol. 111v)</label> Sir Penetrans-domos conducting uroscopy: friar in a brown hooded cloak and pale gown, holding up a diagnostic urine flask (jordan), coloured blue. Possibly meant to depict <persName key="person_173452882">Henry Daniel</persName>: cf. John B. Friedman, ‘<ref target="https://doi.org/10.1484/J.YLS.2.302846">The Friar Portrait in Bodleian Library MS. Douce 104: Contemporary Satire?</ref>’, <title>The Yearbook of Langland Studies</title> 8 (1994): 177–85.</item>
+                        </list>
+                     </decoNote>
+                     <decoNote type="colInit">3-line decorated initials at passus beginnings, blue with red background.</decoNote>
```
Comment (holfordm) [inline collections/Douce/MS_Douce_104.xml:245]: "flourInit" rather than "colInit"?

File: collections/Douce/MS_Douce_104.xml [hunk 4]
```diff
@@ -37,40 +36,238 @@
...
+                     <handNote script="cursivaAntiquior">Cursiva antiquior, written by a single scribe.</handNote>
+                  </handDesc>
                   <decoDesc>
-                     <decoNote type="miniature">Miniatures (coloured drawings). (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 886, pl. LXXXIV)</decoNote>
+                     <decoNote type="miniature">72 marginal miniatures. Miniatures (coloured drawings). (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 886, pl. LXXXIV) Description of miniatures based on the extensive interpretation of <ref target="https://doi.org/10.1484/J.YLS.2.302913">Kathleen Scott, <q>The Illustrations of MS Douce 104</q></ref>: <list>
+                           <item>
+                              <label>(fol. 5r)</label> The Castle of Care (?): pencil sketches only.</item>
+                           <item>
+                              <label>(fol. 8r)</label> Lady Meed: young upright woman wearing a gold crown with a criclet of braids over each ear, bearing three goblets in each hand.</item>
+                           <item>
```
Comment (holfordm) [inline collections/Douce/MS_Douce_104.xml:97]: typo ("criclet")

File: collections/Douce/MS_Douce_104.xml [hunk 5]
```diff
@@ -37,40 +36,238 @@
...
+                              <label>(fol. 15r)</label> Conscience: man with a forked beard, looking away from the text.</item>
+                           <item>
+                              <label>(fol. 18r)</label> The King: man with a forked beard in ermine copy and lavender robe, holding a red sceptre.</item>
+                           <item>
+                              <label>(fol. 19r)</label> Reason (?): seated figure with a short forked beard, wearing a hat and gloves.</item>
+                           <item>
+                              <label>(fol. 23r)</label> Tom Stowe: young man in a cote-hardie (jacket) and hat.</item>
+                           <item>
+                              <label>(fol. 24r)</label> Pride: young man dressed in tights and doubled, white collar, with a belt hung with <q>folly bells</q>, gloves, cuffs, and a jewelled chaperon.</item>
+                           <item>
```
Comment (holfordm) [inline collections/Douce/MS_Douce_104.xml:115]: typo for doublet?

File: collections/Douce/MS_Douce_104.xml [hunk 6]
```diff
@@ -37,40 +36,238 @@
...
+                              <label>(fol. 26v)</label> Lechery: kneeling young man with a sword on his belth, raising both hands in fists above his head.</item>
+                           <item>
+                              <label>(fol. 27r)</label> Avarice: standing bearded man in a knee-length gown, grasping a pouch fastened through his belt, gesturing towards the text.</item>
+                           <item>
+                              <label>(fol. 29r)</label> Gluttony: man holding a bowl to his mouth with both hands, wearing hose and short gown to show a distended stomach and navel.</item>
+                           <item>
+                              <label>(fol. 31r)</label> Sloth: seated young man in a rumpled gown, open at the front, with one bare foot, asleep with his head in his hand.</item>
+                           <item>
+                              <label>(fol. 33b recto)</label> Pilgrim: standing figure of a breaded man, with a broad-brimmed straw hat, a rough hooded outer garment, and a bell on the wrist or waist, holding a staff.</item>
+                           <item>
```
Comment (holfordm) [inline collections/Douce/MS_Douce_104.xml:129]: typo for bearded

**Comment by adunning** [comment]
Thank you for your corrections!

**Pull request merged**